app/assets/stylesheets/general.scss

```
@@ -0,0 +1,7 @@
+.table {
+    border: .5px solid grey;
+
+    th, td {
+        border: .5px solid grey;
+    }
+}
```

```suggestion
}

```

Newline character at the end of the file please!